extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ScenarioActivity.scala

```diff
@@ -283,10 +307,8 @@ object ScenarioActivity {
   ) extends BatchDeploymentRelatedActivity {
 
     override def result: DeploymentResult = scheduledExecutionStatus match {
-      case ScheduledExecutionStatus.Finished                => DeploymentResult.Success(dateFinished)
-      case ScheduledExecutionStatus.Failed                  => DeploymentResult.Failure(dateFinished, None)
-      case ScheduledExecutionStatus.DeploymentWillBeRetried => DeploymentResult.Failure(dateFinished, None)
-      case ScheduledExecutionStatus.DeploymentFailed        => DeploymentResult.Failure(dateFinished, None)
```

Comment:
Why is it not an activity anymore? It looks like important information for the user: we skipped some schedule invocation because of deployment failure.

Reply:
Restored, with fixed date handling (for error statuses there it no starting date available)